src/hooks/useCart/useCart.tsx

```diff
@@ -151,7 +157,7 @@ export const useCart = (
         if (e instanceof NotEligibleError) {
           setCartState("NOT_ELIGIBLE");
           // Cart will remain in FETCHING_QUOTA state.
-        } else if (e instanceof PolicyError) {
+        } else if (e instanceof EnvVersionError) {
           setError(
             new Error(
               "Encountered an issue obtaining policies. We've noted this down and are looking into it!"
```

Comment:
small change here, shall we change the wording to `obtaining environment information`, or `obtaining version info`?

Reply:
okie!